storybookjs/aem
# Add eslint config files (from storybooksjs repo) and run with --fix flag

Issue: Lint configs not added yet

## What I did
Add ESLint config files from [storybook](https://github.com/storybookjs/storybook). Ran yarn lint:js with --fix flag to fix any easy linting issues.

## How to test
Run builds and existing examples and ensure they still work as expected.

- Does this need an update to the documentation?
  - No, will need one after all errors are fixed
